algo/src/main/java/com/hartwig/serve/sources/ckb/ActionableTrialFactory.java

```diff
@@ -54,7 +66,7 @@ public Set<ActionableEntry> create(@NotNull CkbEntry entry, @NotNull String sour
         Set<ActionableEntry> actionableTrials = Sets.newHashSet();
 
         for (ClinicalTrial trial : trialsToInclude(entry)) {
-            Set<String> countries = filterOnRegionsToInclude(trial, regionsToInclude);
+            Set<Country> countries = filterOnRegionsToInclude(trial, regionsToInclude, ckbFacilityCuration);
```

Comment:
Maybe rename this function to be more representative? e.g. `extractCountriesToInclude`?